tests/rt.conf

```diff
@@ -188,3 +188,9 @@ RUN     | datm_mx025_gefs
 
 COMPILE | APP=DATM_NEMS DEBUG=Y                                                                                                   | - wcoss_cray jet.intel                  | fv3 |
 RUN     | datm_debug_cfsr                                                                                                         | - wcoss_cray jet.intel                  | fv3 |
+
+###################################################################################################################################################################################
+# CDEPS Data Atmosphere tests                                                                                                                                                     #
+###################################################################################################################################################################################
+
+COMPILE | APP=DATM                                                                                                                | - wcoss_cray jet.intel                  | fv3 |
```

Comment:
I think we need to have a convention in here. Defining DATM will not make sense when other applications/configurations came. If this configuration is specific to S2S, it might be name it as S2S_DATM etc. It is also same for other app names. I think just by looking APP new you need to cat something about its configuration (global vs. regional, S2S vs. HAFS etc.)

Reply:
The UFS weather model has a single data component. This will have to be discussed when someone opens a PR to add others. 
Is this a blocking comment for getting the build of CDEPS into the UFS weather model?

Reply:
No, that was just a suggestion. In any case, you will review the PR coming from HAFS and we could make modification in there based on your comments. I am approving this PR.